.github/actions/spell-check/expect/expect.txt

```diff
@@ -203,6 +203,8 @@ CARETBLINKINGENABLED
 CARRIAGERETURN
 cascadia
 catid
+carlos
+zamora
```

Comment:
Don't we have a names exception file?

Reply:
I'm already in names.txt. Idk what's going on.